Automattic/compare-snapshot-tool
# Add initial changes

## Changes 

This PR is the initial implementation of the side-by-side tool. This covers the following functionalities: 

1. View and compare the URLs from the CSV file 
2. Update status as passed or failed 
3. Download a new CSV file with new statuses

**NOTE: You always need to download the file to save the status.** 

## Testing 
I would appreciate more eyes on the testing! You can use this CSV file as your test data: [posts-urls copy.csv](https://github.com/Automattic/compare-snapshot-tool/files/10386519/posts-urls.copy.csv). Feel free to add URLs in the `newUrl` column.

- Run `yarn` to install dependencies.
- Run `yarn start` to see the development build.

## Feedback needed
- This is my attempt at a React app! I will appreciate feedback on the structure and how I can break down the components into more readable ones. 
- Any scenarios that you think should be covered in this tool? 

## What's next 
- [ ] Cleanup the files
